test/fs.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const metatests = require('metatests');
+const fs = require('fs');
+const { rmdirp } = require('..');
+
```

Comment:
The test directory path should be moved in the constant at the beginning and I think it should be `test/testDir`, not just `testDir`. Also, we might support Windows platform in the future, so it will be more convenient to use `path.sep` or `path.join` instead of hardcoding.
```suggestion
const RMDIRP_TEST_DIR = `test${path.sep}testDir`;
```